interview-questions.md

```diff
@@ -4,17 +4,19 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: Its used to call back to parent objects, and is easy to recognize.
 
-  Researched answer:
+  Researched answer: The method super is used to refer to parent objects and to access the superclass constructor,
+  it used to get rid of confusion between superclasses and subclasses that may have the same name.
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer:It's a virtual representation of a UI.
```

Comment:
Somewhat, but again this is a pretty simplistic answer. I'd encourage you to take time and try to explain things in a bit deeper detail. Pretend this is your DREAM JOB asking you this question, would you be really comfortable with that answer?